cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -1032,6 +1035,73 @@ pub enum ManifestBuildSandbox {
     Pure,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Default)]
+#[serde(deny_unknown_fields)]
+#[cfg_attr(test, derive(proptest_derive::Arbitrary))]
+pub struct ManifestContainerize {
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub config: Option<ManifestContainerizeConfig>,
+}
+
+/// Container config derived from
+/// https://github.com/opencontainers/image-spec/blob/main/config.md
+///
+/// Env and Entrypoint are left out since they interfere with our activation implementation
+/// Deprecated and reserved keys are also left out
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Default)]
+#[serde(rename_all = "PascalCase")]
```

Comment:
Suggestion: I think matching the case of the OCI options is going to look out of place in the manifest. Everything else is kebab-case. Was there discussion around that at all? I wouldn't think that we would adopt the casing of other tools if we added them.

Reply:
I think it's a little weird but also emphasizes we're just following the OCI spec. I don't know how much it was discussed but it's what's in the ticket https://github.com/flox/flox/issues/1260

Reply:
Since the manifest format is something we're unlikely to change the format of without a very good reason, I'd like to confirm that this is the intention and not something that just got put into the ticket.